jstests/tests/test_analytics.js

```diff
@@ -4,14 +4,24 @@ define(function(require) {
     
     QUnit.module("Google Analytics");
     QUnit.test("test without analytics", function (assert) {
+        var result=[];
+        window.ga = function(cmd, id, auto) {
+            result[0] = [cmd, id, auto];
+        };
         window.apidata.analytics = undefined;
         var ga = analytics.init();
```

Comment:
You're not using `ga` variable after that, just `analytics.init();` is enough I think.